tests/conftest.py

```diff
@@ -1,27 +1,12 @@
-# For easy inspection on what dependencies were used in test.
-def pytest_report_header(config):
-    import sys
+from pytest_astropy_header.display import (PYTEST_HEADER_MODULES,
+                                           TESTED_VERSIONS)
 
-    s = "\nFull Python Version: \n{0}\n\n".format(sys.version)
 
-    try:
-        import warnings
-        from astropy.utils.introspection import resolve_name
-    except ImportError:
-        return s
+def pytest_configure(config):
+    PYTEST_HEADER_MODULES.pop('h5py')
+    PYTEST_HEADER_MODULES.pop('Pandas')
```

Comment:
`h5py` and `pandas` are not package dependencies, so probably don't need to be here?

Reply:
They are included by default in `pytest-astropy-header` plugin, so this is actually to remove them so they won't show up in our test header.

Reply:
Doh!  Carry on.

Reply:
In that case, maybe remove `scipy` and `matplotlib` too, as those seem to be included by default.

Reply:
Done.